packages/env/lib/wordpress.js

```diff
@@ -55,7 +55,10 @@ async function configureWordPress( environment, config, spinner ) {
 		return `${ domain }:${ port }`;
 	} )();
 
-	const installCommand = `wp core install --url="${ url }" --title="${ config.name }" --admin_user=admin --admin_password=password --admin_email=wordpress@example.com --skip-email`;
+	const email = config.env[ environment ].config.WP_TESTS_EMAIL ? config.env[ environment ].config.WP_TESTS_EMAIL : 'wordpress@example.com';
```

Comment:
I think we can slightly simplify this. In the config file, there are already default values:

https://github.com/WordPress/gutenberg/blob/53977c637d766d2c41aa99cf0290cee05763d3e0/packages/env/lib/config/config.js#L84-L85

I think the problem is just that we aren't using them for the installation. But they should still be set as constants in the WordPress environment.

That said, the "TESTS_EMAIL" and "TESTS_TITLE" are the same for all environments. Which is a little counter-intuitive, but they do have "tests" in the name. So I don't think we should use them for the title and email of the non-test environments.. But another caveat is that I'm not sure how those constants are meant to be used in the first place. 

The main reason I want to be cautious here is that we'd be changing the admin email for all existing installations. I'd like to avoid that because it seems like a breaking change. 

And the title change would also break the `config.name` feature. 🤔 

This might be a bad assumption, but _if_ the `WP_TESTS_*` constants are more for internal WordPress stuff, does it make more sense to do the following instead?
- Allow the "name" to be different in each environment.
- Add an option for setting the "admin email"

However, after saying all of that, could you clarify the use case you have for this feature?

Reply:
Nice! I think adding a wp-env config option would be the best path forward -- one that can be changed in different environments. We can then use that to set both the TESTS_* constants as well as the installation parameters